TTS/tts/models/vits.py

```diff
@@ -283,16 +327,22 @@ def __init__(self, config: Coqpit, speaker_manager: SpeakerManager = None):
 
         if args.use_sdp:
             self.duration_predictor = StochasticDurationPredictor(
-                args.hidden_channels,
+                args.hidden_channels + self.embedded_language_dim,
                 192,
                 3,
                 args.dropout_p_duration_predictor,
                 4,
                 cond_channels=self.embedded_speaker_dim,
+                language_emb_dim=self.embedded_language_dim,
             )
         else:
             self.duration_predictor = DurationPredictor(
-                args.hidden_channels, 256, 3, args.dropout_p_duration_predictor, cond_channels=self.embedded_speaker_dim
+                args.hidden_channels + self.embedded_language_dim,
+                256,
+                3,
+                args.dropout_p_duration_predictor,
+                cond_channels=self.embedded_speaker_dim,
+                language_emb_dim=self.embedded_language_dim,
```

Comment:
Why do we pass. `args.hidden_channels + self.embedded_language_dim`, if we already pass                 `language_emb_dim=self.embedded_language_dim`,

Reply:
Done